package.json

```diff
@@ -27,10 +27,16 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --watchAll --coverage",
```

Comment:
nice stuff! if you have a code amount of code coverage in this repository, you should highlight that and show it off in the README so people know that this code is not only solid but it's been tested up to a specific coverage percentage.